tests/E2E Tests/WebAppUiTests/B2CWebAppCallsWebApiLocally.cs

```diff
@@ -46,7 +49,7 @@ public async Task Susi_B2C_LocalAccount_TodoAppFucntionsCorrectly()
         {
             // Web app and api environmental variable setup.
             DefaultAzureCredential azureCred = new();
-            string clientSecret = await UiTestHelpers.GetValueFromKeyvaultWitDefaultCreds(_keyvaultUri, KeyvaultClientSecretName, azureCred);
+            string clientSecret = await UITH.GetValueFromKeyvaultWitDefaultCreds(_keyvaultUri, KeyvaultClientSecretName, azureCred);
```

Comment:
I suggest keeping `UiTestHelpers` naming. `UITH` is not a common acronym, so it's not immediately obvious what it represents - makes code less readable. Plus with IntelliSense auto-complete, the name length is not a huge issue.